modules/installation-aws-editing-manifests.adoc

```diff
@@ -0,0 +1,126 @@
+//
+// * installing/installing_aws/installing-aws-outposts-remote-workers.adoc
+
+:_content-type: PROCEDURE
+[id="installation-aws-creating-manifests_{context}"]
+= Generating manifest files
+
+Use the installation program to generate a set of manifest files in the assets directory.
+
+If you plan to reuse the `install-config.yaml` file, create a backup file before you generate the manifest files.
+
+.Procedure
+
+. Optional: Create a backup copy of the `install-config.yaml` file:
++
+[source,terminal]
+----
+$ cp install-config.yaml install-config.yaml.backup
+----
+
+. Generate a set of manifests in your assets directory:
++
+[source,terminal]
+----
+$ openshift-install create manifests --dir <installation_-_directory>
+----
++
+This command displays the following messages.
++
+.Example output
+[source,terminal]
+----
+INFO Consuming Install Config from target directory
+INFO Manifests created in: <installation_directory>/manifests and <installation_directory>/openshift 
+----
++
+The command generates the following manifest files:
++
+.Example output
+[source,terminal]
+----
+$ tree
+.
+├── manifests
+│   ├── cluster-config.yaml
+│   ├── cluster-dns-02-config.yml
+│   ├── cluster-infrastructure-02-config.yml
+│   ├── cluster-ingress-02-config.yml
+│   ├── cluster-network-01-crd.yml
+│   ├── cluster-network-02-config.yml
+│   ├── cluster-proxy-01-config.yaml
+│   ├── cluster-scheduler-02-config.yml
+│   ├── cvo-overrides.yaml
+│   ├── kube-cloud-config.yaml
+│   ├── kube-system-configmap-root-ca.yaml
+│   ├── machine-config-server-tls-secret.yaml
+│   └── openshift-config-secret-pull-secret.yaml
+└── openshift
+    ├── 99_cloud-creds-secret.yaml
+    ├── 99_kubeadmin-password-secret.yaml
+    ├── 99_openshift-cluster-api_master-machines-0.yaml
+    ├── 99_openshift-cluster-api_master-machines-1.yaml
+    ├── 99_openshift-cluster-api_master-machines-2.yaml
+    ├── 99_openshift-cluster-api_master-user-data-secret.yaml
+    ├── 99_openshift-cluster-api_worker-machineset-0.yaml
+    ├── 99_openshift-cluster-api_worker-user-data-secret.yaml
+    ├── 99_openshift-machineconfig_99-master-ssh.yaml
+    ├── 99_openshift-machineconfig_99-worker-ssh.yaml
+    ├── 99_role-cloud-creds-secret-reader.yaml
+    └── openshift-install-manifests.yaml
+
+----
+
+[id="installation-aws-editing-manifests_{context}"]
+== Modifying manifest files
+
+[NOTE]
+====
+The AWS Outposts environments has the following limitations which require manual modification in the manifest generated files:
+
+* The maximum transmission unit (MTU) of a network connection is the size, in bytes, of the largest permissible packet that can be passed over the connection. AWS Outposts requires a minimum of 1500 bytes across your on-premises network. The Outpost service link supports a maximum packet size of 1300 bytes. For more information about the service link, see  link:https://docs.aws.amazon.com/outposts/latest/userguide/region-connectivity.html[Outpost connectivity to AWS Regions]
```

Comment:
This information is confusing. I can't understand the relationship between the 1500 to 1300 limitations.

Reply:
```suggestion
* The maximum transmission unit (MTU) of a network connection is the size, in bytes, of the largest permissible packet that can be passed over the connection. The Outpost service link supports a maximum packet size of 1300 bytes. For more information about the service link, see  link:https://docs.aws.amazon.com/outposts/latest/userguide/region-connectivity.html[Outpost connectivity to AWS Regions]
```

Reply:
Removed